src/settings/toml/environment.rs

```diff
@@ -3,7 +3,7 @@ use super::site::Site;
 
 use serde::{Deserialize, Serialize};
 
-#[derive(Clone, Debug, Deserialize, Serialize)]
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
```

Comment:
added to allow comparison of Manifests for the benefit of testing constructor methods against one another.